### DISCUSSION TITLE
The server has reached its maximum number of sessions

### DISCUSSION THREAD

**Discussion by hell0w0orld** [eclipse-milo/milo#1184] [General]
I encountered this issue when connecting to an opcUaServer embedded in the device. How should I solve it?

**Comment by kevinherron** [comment]
Figure out how many sessions the server allows, and what other OPC UA client applications have open sessions, or why you have so many open.

**Reply by hell0w0orld** [reply]
Thank you for your suggestion. I have resolved this issue

**Reply by WendelYu** [reply]
大佬怎么解决的
